3457/1/parser.rb

```diff
@@ -0,0 +1,43 @@
+require 'optparse'
+# Class CommandLineParser is responsible for parsing
+class CommandLineParser
```

Comment:
Why do you use only class method in that class? It's not the best approach in your case